test/test_cli.rb

```diff
@@ -40,6 +40,23 @@ def test_invalid_options_ignored_recursive_current_dir
     assert_equal %w[.], options.files
   end
 
+  def test_options_file_without_required
+    option_file_path = File.expand_path('../fixtures/extra-options.txt', __FILE__)
+    err = assert_raise(OptionParser::InvalidOption) do
+      RipperTags.process_args(['--options', option_file_path])
+    end
+    assert_equal "invalid option: needs either a list of files, `-L`, or `-R' flag", err.message
+  end
+
+  def test_options_file
+    option_file_path = File.expand_path('../fixtures/extra-options.txt', __FILE__)
+    options = process_args(['--options', option_file_path, '-R'])
```

Comment:
The test should also verify:

1. that multiple `--options=FILE` or `--options FILE` are supported per single invocation;
2. that arguments read from FILE are inserted into the appropriate position within the rest of the parameters (we could verify this with the order things appear in `exclude` list);
3. that unsupported options are allowed when `--options` is combined with `--ignore-unsupported-options`.